nevergrad/optimization/test_optimizerlib.py

```diff
@@ -438,22 +438,20 @@ def _ellips(x: np.ndarray) -> float:
     contextual_budget *= int(max(1, np.sqrt(scale)))
 
     # Let us run the comparison.
-    default = (registry["CMA"] if dimension > 1 else registry["OnePlusOne"])(dimension, contextual_budget, num_workers=num_workers)
-    MetaModel = registry["MetaModel"](dimension, contextual_budget, num_workers=num_workers)
-    default_recom, MetaModel_recom = default.minimize(_target), MetaModel.minimize(_target) 
-    default_data = default_recom.get_standardized_data(reference=default.parametrization)
-    MetaModel_data = MetaModel_recom.get_standardized_data(reference=MetaModel.parametrization)
+    default = registry["CMA" if dimension > 1 else "OnePlusOne"](dimension, contextual_budget, num_workers=num_workers)
+    metamodel = registry["MetaModel"](dimension, contextual_budget, num_workers=num_workers)
+    default_recom, metamodel_recom = default.minimize(_target).value, metamodel.minimize(_target).value 
```

Comment:
Multiple long statements on a same line is very hard to read. Also, this is a lot of repeated code
```suggestion
recommendations: tp.List[np.ndarray] = []
for name in ("MetaModel", "CMA" if dimension > 1 else "OnePlusOne"):
    opt = registry[name](dimension, contextual_budget, num_workers=num_workers)
    recommendations.append(opt.minimize(_target).value)
metamodel_recom, default_recom = recommendations
```